### ISSUE TITLE
Does it support Nano SAM ?

### ISSUE THREAD

**Issue by Rocky-YU** [zhudongwork/SAM_TensorRT#8]
Nice work!
In the quest for faster speeds, is the framework applicable to Nano SAM ?

**Comment by zhudongwork** [comment]
Yes, Nano Sam does not seem to have any weird ops. The framework should be able to support it. You just need to adjust the input and output parts of the code.

**Comment by Rocky-YU** [comment]
Ok, thank you very much for your reply, I'll try it.
